ui/shared/components/buttons/SearchResultsLink.jsx

```diff
@@ -48,15 +48,14 @@ const ConnectedSearchResultsLink = connect(null, mapDispatchToProps)(SearchResul
 
 export default ConnectedSearchResultsLink
 
-const getGeneSearchProps = af => ({
-  inheritance: { mode: ANY_AFFECTED },
-  freqs: FREQUENCIES.filter(({ name }) => name !== THIS_CALLSET_FREQUENCY && name !== SV_CALLSET_FREQUENCY).reduce(
-    (acc, { name }) => ({ ...acc, [name]: { af } }), {},
-  ),
-})
-
-const INITIAL_GENE_SEARCH = getGeneSearchProps(0.1)
-const RARE_GENE_SEARCH = getGeneSearchProps(0.01)
+const INITIAL_GENE_SEARCH = {
+  inheritance: { mode: ANY_AFFECTED, filter: {} },
+  freqs: {
+    ...FREQUENCIES.filter(({ name }) => name.startsWith('gnomad_') || name === THIS_CALLSET_FREQUENCY).reduce(
```

Comment:
per my review, we should be filtering out topmed, not filtering for literally everything else

Reply:
Do you mean all freq filters other than topmed have a threshold of 0.03?

Reply:
yes